client/java/src/test/resources/config/datasetNamespaceResolver.yaml

```diff
@@ -0,0 +1,16 @@
+transport:
+  type: console
+dataset:
+  namespaceResolvers:
```

Comment:
Looking at those examples, I wonder if we should not add optional field indicating `schema` of the namespace that the resolver would be operating on. 

For example, I would like to specify that `kafka-prod` resolver would work only on `kafka` namespaces, while `cassandra-prod` would work on `cassandra` namespaces.

I think there exists a risk of mixing those up, especially with too wide regex.